src/content/concepts/entry-points.md

```diff
@@ -8,6 +8,8 @@ contributors:
   - sokra
   - EugeneHlushko
   - Zearin
+  - niravasher
```

Comment:
I couldn't find any contribution from `niravasher` in this file. 
please check this. maybe I am missing something.